### ISSUE TITLE
Strange Behaviour when embedded in NavigationStack

### ISSUE THREAD

**Issue by muthenberg** [hCaptcha/HCaptcha-ios-sdk#113]
## Bug description
I try to add an HCaptchaView following the example to an iOS App build using SwiftUI. I've got a LoginScreen that is a NavigationStack and leads to a RegistrationScreen upon user request. The RegistrationScreen is embedded inside the NavigationStack from the LoginScreen, so the user can easily navigate back. The problem is, that the challenge takes forever to appear as long as I have that NavigationStack around everything. As soon as I remove the NavigationStack everything works fine.

## To Reproduce
1. Add a NavigationStack around the HCaptcha view (for example as part of the example application)
2. Run the app and try to get a token
3. It will take several minutes until the challenge appears (if at all).

## Expected behavior
It should behave inside a NavigationStack the same way as outside of it

## Context
I am using iOS 16.2 and version 2.5.0 of this library.

**Comment by e271828-** [comment]
Thanks for the report, we're taking a look at this.

**Comment by CAMOBAP** [comment]
@muthenberg thanks for reporting

I have tried to reproduce it with:

```
--- a/Example/SwiftUI/App.swift
+++ b/Example/SwiftUI/App.swift
@@ -8,10 +8,13 @@
 import SwiftUI

 @main
+@available(iOS 16.0, *)
 struct HCaptchaSwiftUIExampleApp: App {
     var body: some Scene {
         WindowGroup {
-            HCaptchaView()
+            NavigationStack {
+                HCaptchaView()
+            }
         }
     }
 }
```

But SDK works correctly, maybe I missed some important aspect?
